### DISCUSSION TITLE
Pitch knob transposition in semitones?

### DISCUSSION THREAD

**Discussion by DepreTux** [bangcorrupt/hacktribe#51] [Ideas]
Would that be feasible?
Either that or any other way to transpose the patterns quickly would vastly improve the performing capabilities of the electribe, I think.

More importantly, thanks for this. I can't believe my electribe just got a lot more useful with all the features!

Have a happy new year!

**Comment by bangcorrupt** [comment]
> way to transpose the patterns quickly

It's not planned any time soon, but keep an eye on #59.

**Reply by DepreTux** [reply]
> > way to transpose the patterns quickly
> 
> It's not planned any time soon, but keep an eye on #59.

So I can dump the pattern, transpose it, and upload it again? Great!
Thanks!

**Comment by egasimus** [comment]
Pitch knob in semitones would be awesome. Right now it's easy to transpose +2 or -2 octaves but not by +/-1 (the knob goes -63 to +63, so the right pitch would be 63/2=31.5?)

**Reply by bangcorrupt** [reply]
48 is one octave.  It hits some exact semitones on the way, I can't remember which but it's probably the same harmonics I used for the FM oscillators.

Octave range and curve type would be good options for the pitch knob.
